_docs/backenders/strategy-2021/roles/hiring.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Template
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Capacity role definition
+
+**Current responsible**: -
+
+
+
+***Note:***
+
+2 people Identify the hiring related tasks to be managed between different roles:
+- Capacity
+- Strategy
+- Mentorship Coordination
+
+## Responsibilities
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
+          
+          Priority: HIGH priority for all   
+          Identified as responsibility to be shared with Capacity Role 
+
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions 
+  
+          Priority: HIGH priority for all   
+          Identified as responsibility to be shared with Capacity, Strategy, M.C roles
+
+
+- Distributing tasks: reviewing candidates, attending interviews
+        
+          Priority: Average Medium priority. (2 Low, 1 Medium, 1 High)
+          Agn and Andr are happy with the way that Sebastion is managing his part of this responsibility
```

Comment:
keyboards are cheap, no need to save on characters typed :P please use full names